test/e2e_node/topology_manager_test.go

```diff
@@ -313,6 +328,36 @@ func validatePodAlignment(f *framework.Framework, pod *v1.Pod, envInfo *testEnvI
 	}
 }
 
+// validatePodAligmentWithPodScope validates whether all pod's CPUs are affined to the same NUMA node.
+func validatePodAlignmentWithPodScope(f *framework.Framework, pod *v1.Pod, envInfo *testEnvInfo) error {
```

Comment:
Have you verified that these actually run? I know you have to send a specific command in the PR comments to trigger them (they are not automatic as far as I know).

Reply:
/cc @vpickard for more information

Reply:
/test pull-kubernetes-node-kubelet-serial-topology-manager

Reply:
@klueska Yes, the jobs still have to be manuall triggered, as above.

Reply:
We were verifying tests in our test environment. As I see tests are passing here too. Thanks for the command Victor!